prisma/schema.prisma

```diff
@@ -11,11 +11,12 @@ datasource db {
 }
 
 model User {
-  id       Int      @id @default(autoincrement())
-  email    String   @unique
-  password String
-  role     Role     @default(USER)
-  recipes  Recipe[]
+  id                  Int                   @id @default(autoincrement())
+  email               String                @unique
+  password            String
+  role                Role                  @default(USER)
+  recipes             Recipe[]
+  personalAccessToken PersonalAccessToken[]
```

Comment:
hmm, I wouldn't add `personalAccessToken` to the `User` table. You already have `userId` in `PersonalAccessToken` table - this should be enough.

Reply:
I can't remove `personalAccessToken` here.
Prisma throws this error:
```
Error validating field 'user' in model 'PersonalAccessToken': 
The relation field 'user' on model 'PersonalAccessToken' is missing an opposite relation field on the model 'User'`
```